control-base/src/main/java/fi/nls/oskari/control/data/GetGeoPointDataHandler.java

```diff
@@ -82,8 +82,12 @@ public void handleAction(final ActionParameters params) throws ActionException {
 
 			final OskariLayer layer = mapLayerService.find(layerId);
 			final String layerType = layer.getType();
-
-			if (OskariLayer.TYPE_WMS.equals(layerType)) {
+			//HSY doesn't want GFI from basemaps.
+			boolean isBaseMap = false;
```

Comment:
And a quick fix for the previous commit: https://github.com/oskariorg/oskari-server/commit/3e7a95d99814253da68af02373111e61c0e75691